guide/ops/troubleshooting/memory-usage.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: website-normal
+title: "Troubleshooting: Monitoring Memory Usage"
+toc: /guide/toc.json
+---
+
+## Memory Usage
```

Comment:
Worth adding a link to this page from https://github.com/apache/brooklyn-docs/blob/master/guide/ops/server-cli-reference.md#memory-usage